gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/ExplodeBundle.java

```diff
@@ -37,6 +38,14 @@
         this.entityLinkerRegistry = entityLinkerRegistry;
     }
 
+    void verifyExistingFolderPath(Bundle bundle, String folderPath) {
+        if (folderPath.equals("/")) {
+            return;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this redundant jump. ([squid:S3626](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS3626))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=131&issues=AWcucRmYKqTTJP1BKO56&open=AWcucRmYKqTTJP1BKO56)

[ISSUE_KEY]: AWcucRmYKqTTJP1BKO56